Gemfile

```diff
@@ -16,7 +16,11 @@ else
   gem 'gds-api-adapters', '25.1.0'
 end
 
-gem 'govuk_message_queue_consumer', '~> 2.1.0'
+if ENV["MESSAGE_QUEUE_CONSUMER_DEV"]
```

Comment:
I don't want to edit this file to change the version number to the gem path, and then run `bundle update govuk_message_queue_consumer`

instead I just have to write `MESSAGE_QUEUE_CONSUMER_DEV=true bundle update govuk_message_queue_consumer`

Unless you know of a better way that avoids editing this file when developing a gem I rather leave this here.